apps/web-giddh/src/app/ledger/ledger.component.html

```diff
@@ -991,7 +991,7 @@ <h3 class="ledgerHead">
                                         {{ ledgerTxnBalance?.currencySymbol }}{{lc.reckoningDebitTotal  | giddhCurrency }}
                                     </span>
 
-                                    <span class="bdrT ng-binding" style="padding: 1px 0;"
+                                    <span class="bdrT ng-binding text-gray" style="padding: 1px 0;"
```

Comment:
remove inline CSS

Reply:
I am not adding this inline css, we can see on green line text-gray classes is added because dark green is showing on this class.